config/label-management.md

```diff
@@ -0,0 +1,126 @@
+# Managing cross-organization labels in existing repositories #
+
+This is a step-by-step guide/script for managing organization-wide GitHub
+issue labels across all repositories.
+
+## Prerequisites ##
+
+- User has access to the GitHub organization's repositories
+- User can generate a GitHub [Personal access token](https://github.com/settings/tokens)
+
+## Initial Setup ##
+
+1. Clone the repository locally
+
+    ```console
+    git clone https://github.com/alta3/github-label-management.git
+    cd github-label-management
+    ```
+
+1. Activate/create the virtual environment and install the requirements
+
+    ```console
+    pyenv local 3.8.3
+    pyenv virtualenv github-label-management
+    pyenv activate github-label-management
+    python3 -m pip install --upgrade pip setuptools wheel
+    python3 -m pip install --requirement requirements.txt
+    ```
+
+1. Create a local configuration file by running `touch config`
+1. Open `config` in your favorite editor and start from this template
+
+    ```sh
+    export GITHUB_USER_TOKEN="<YOUR_TOKEN_HERE>"
+    export LABELS_TOKEN=${GITHUB_USER_TOKEN}
+    export LABELS_USERNAME="<YOUR_USERNAME>"
+    export LABELS_ORGNAME="cisagov"
+    ```
+
+1. Create a GitHub [personal access token](https://github.com/settings/tokens)
+    - Give it a descriptive name like "label management"
+    - **Only** select the `repo` scope
+    - Copy the generated token into `GITHUB_USER_TOKEN`
+1. Save your changes and run `source config`
+
+## Label management operations ##
+
+There are several operations you'll probably want to do, now that you've
+set up the environment and have the script ready.
+
+### Re-initializing the environment ###
+
+When you return to the directory in a new shell, remember to:
+
+```console
+pyenv activate github-label-management
```

Comment:
If you've set up `pyenv` as we instruct in our `CONTRIBUTING.md` files ([cisagov/skeleton-generic's as an example](https://github.com/cisagov/development-guide/blob/develop/CONTRIBUTING.md#installing-and-using-pyenv-and-pyenv-virtualenv)), then this step should not be necessary as long as the correct virtualenv has been set with `pyenv local`.

Reply:
This will be resolved by moving to a skeletonized version in https://github.com/cisagov/github-label-management.